### PR TITLE
OPSEXP-1766: fix ES protocol Issue

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1766

### CODE REVIEW

File: helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {{- define "repo.elasticsearch.config" -}}
 -Dindex.subsystem.name=elasticsearch
+{{- if eq (index .Values "alfresco-elasticsearch-connector" "elasticsearch" "protocol" | default .Values.global.elasticsearch.protocol) "https" }}
+-Delasticsearch.secureComms=https
+{{- end }}
```
Comment (aborroy) [inline helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl:5]: In addition to this environment variable, `encryption.ssl.truststore.*` values should include also the public certificate from the Elasticsearch server TLS protocol.

**Review by aborroy — CHANGES_REQUESTED** [review]
Adding truststore configuration.

File: helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl (hunk 1 above)
Comment (gionn) [reply helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl:5]: why is that needed? our images should have proper cacerts and ssl certificates used by aws opensearch endpoints should already be recognized as trusted

**Review by aborroy — APPROVED** [review]
LGTM

File: helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl (hunk 1 above)
Comment (aborroy) [reply helm/alfresco-content-services/templates/_helpers-elasticsearch.tpl:5]: You're right @gionn 
Repository is using a custom truststore to communicate with SOLR, but for Elasticsearch standard REST Client is used (so default JVM `cacerts` is enough).

**Pull request merged**